docs/DeveloperGuide.md

```diff
@@ -646,6 +646,45 @@ testers are expected to do more *exploratory* testing.
        
     1. Other variations to use after the prefix n/: `///`, `?#$%`, `...`, any other non alphanumeric characters.<br>
        Expected: Similar to previous.
+
+### Editing a passenger
+
+1. Editing passengers with 1 parameter while all passengers shown.
+
+    1. Prerequisites: Using sample passengers, list all passengers using the `list` command. Multiple passengers in the list.
```

Comment:
Following Terence's comments on the other manual test instructions, maybe adding that this is assuming the user is working with the default addressbook.json file?